ClosedXML/Excel/CalcEngine/Functions/MathTrig.cs

```diff
@@ -118,7 +118,7 @@ private static object Cosh(List<Expression> p)
 
         private static object Csch(List<Expression> p)
         {
-            if (p[0].Evaluate().Equals(0))
+            if (p[0].Evaluate().Equals(0.0))
```

Comment:
For floating point comparison, use `if (Math.Abs(p[0].Evaluate()) < Double.Epsilon)`

Reply:
I agree in principle, but for 0 this should be save. If you want I'll change it of course.

Reply:
You can't predict how the floating point rounding will work, e.g. `=CSCH(8-64/8)` . Please change it.